ADMINS-FOR-ISTIO.md

```diff
@@ -6,21 +6,21 @@ access.
-| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for *istio.io | [jasonall](https://github.com/jasonall)
-| [Google Custom Search Engine](https://cse.google.com/cse/) | Search engine for *istio.io | [jasonall](https://github.com/jasonall), [rshriram](https://github.com/rshriram)
-| [Google Domains](https://domains.google.com) | Domain registration for *istio.io | [jasonall](https://github.com/jasonall)
+| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Custom Search Engine](https://cse.google.com/cse/) | Search engine for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Domains](https://domains.google.com) | Domain registration for *istio.io | [craigbox](https://github.com/craigbox)
 | [Google Groups](https://googlegroups.com) | Mailing lists | [craigbox](https://github.com/craigbox)
-| [Google Search Console](https://search.google.com/search-console/about) | Search engine analytics for *istio.io | [jasonall](https://github.com/jasonall)
-| [Netlify](https://netlify.com) | Site hosting for *istio.io | [jasonall](https://github.com/jasonall)
+| [Google Search Console](https://search.google.com/search-console/about) | Search engine analytics for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Netlify](https://netlify.com) | Site hosting for *istio.io | [davidhauck](https://github.com/davidhauck)
 | [Prow](https://prow.istio.io) | Primary CI system | [fejta](https://github.com/fetja),  [cjwagner](https://github.com/cjwagner)
-| [Shared Google Drive](https://drive.google.com/corp/drive/u/0/folders/0AIS5p3eW9BCtUk9PVA) | Design document storage | [rshriram](https://github.com/rshriram), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson), [jasminejaksic](https://github.com/jasminejaksic)
+| [Shared Google Drive](https://drive.google.com/corp/drive/u/0/folders/0AIS5p3eW9BCtUk9PVA) | Design document storage | [craigbox](https://github.com/craigbox), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson), [jasminejaksic](https://github.com/jasminejaksic)
 | [Slack](https://istio.slack.com) | Chat platform | [craigbox](https://github.com/craigbox), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
-| [Twitter](https://twitter.com/IstioMesh) | Istio Twitter account | [oaktowner](https://github.com/oaktowner), [linsun](https://github.com/linsun), [rvennam](https://github.com/rvennam)
+| [Twitter](https://twitter.com/IstioMesh) | Istio Twitter account | [craigbox](https://github.com/craigbox), [linsun](https://github.com/linsun), [rvennam](https://github.com/rvennam)
 | [YouTube](https://www.youtube.com/c/istio) | Working group videos | [craigbox](https://github.com/craigbox)
```

Comment:
macruzbar

Reply:
Add or replace you?

Reply:
I still have access but I don't actively do anything; questions to Maria first.  Maybe list us both in that order.